content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,285 @@
+    ---
+    apiVersion: networking.istio.io/v1alpha3
+    kind: EnvoyFilter
+    metadata:
+      name: filter-ratelimit-svc
+      namespace: istio-system
+    spec:
+      workloadSelector:
+        labels:
+          istio: ingressgateway
+      configPatches:
+        - applyTo: VIRTUAL_HOST
+          match:
+            context: GATEWAY
+            routeConfiguration:
+              vhost:
+                name: "*:80"
+                  route:
+                    action: ANY
+          patch:
+            operation: MERGE
+            # Applies the rate limit rules.
+            value:
+              rate_limits:
+                - actions: # any actions in here
+                  - request_headers:
+                      header_name: ":path"
+                      descriptor_key: "PATH"
+    EOF
+    {{< /text >}}
+
+### Local Rate Limit
+
+Envoy supports using Local Rate Limit on connection and HTTP level. This helps applying rate limit at the instance level
+itself.
+
+You can enable local rate limit by applying Envoy Filter patch as follows. Following enables 100 requests through an
+ istio ingressgateway instance in 60 seconds.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: EnvoyFilter
+    metadata:
+      name: filter-local-ratelimit-svc
+      namespace: istio-system
+    spec:
+      workloadSelector:
+        labels:
+          istio: ingressgateway
+      configPatches:
+        - applyTo: HTTP_FILTER
+          listener:
+            filterChain:
+              filter:
+                name: "envoy.http_connection_manager"
+          patch:
+            operation: INSERT_BEFORE
+            value:
+              name: envoy.filters.http.local_ratelimit
+              typed_config:
+                "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+                type_url: type.googleapis.com/envoy.extensions.filters.http.local_ratelimit.v3.LocalRateLimit
+                value:
+                  stat_prefix: http_local_rate_limiter
+                  token_bucket:
+                    max_tokens: 100
+                    tokens_per_fill: 100
+                    fill_interval: 60s
+                  filter_enabled:
+                    runtime_key: local_rate_limit_enabled
+                    default_value:
+                      numerator: 100
+                      denominator: HUNDRED
+                  filter_enforced:
+                    runtime_key: local_rate_limit_enforced
+                    default_value:
+                      numerator: 100
+                      denominator: HUNDRED
+                  response_headers_to_add:
+                    - append: false
+                      header:
+                        key: x-local-rate-limit
+                        value: 'true'
+    EOF
+    {{< /text >}}
+
+You can also enable local rate limiting for a specific route and it can be achieved as follows:
```

Comment:
Explain a little about what is different from the previous one.

Reply:
Done